gobblin-core/src/main/java/gobblin/source/extractor/utils/Utils.java

```diff
@@ -53,7 +53,28 @@ public static String getClause(String clause, String datePredicate) {
     }
     return retStr;
   }
-
+  
+  /**
+   * Get coalesce of columns if there are multiple comma-separated columns
+   *
+   * @param column
+   * @return column
+   */
+  public static String getCoalesceColumnNames(String column) {
+    int columnCount = 0;
```

Comment:
@Chavdar
Thanks for your comments! I just wanted to touch the least possible code but I agree this method should be optimized.